registry/quilt_server/auth.py

```diff
@@ -272,15 +272,17 @@ def revoke_user_code_tokens(user):
         db.session.delete(code)
     revoke_tokens(user)
 
-def get_exp(mins=30):
-    return datetime.utcnow() + timedelta(minutes=mins)
+def get_exp(**kwargs):
```

Comment:
semantically, this is not a `get` method. it is really `calc_exp` or something like that.